package.json

```diff
@@ -6,6 +6,7 @@
     "@uidotdev/usehooks": "^2.1.0",
     "react-content-loader": "^6.2.1",
     "react-loading-skeleton": "^3.3.1",
+    "react-range-slider-input": "^3.0.7",
```

Comment:
I think this, along with the changes in `package-lock.json` might be in the wrong ones (I ran `npm i` but still get error that the slider component cannot be found). You might need to run the install in the `munch-match` directory so it updates the right files.